src/test/credentials/sso/model.test.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as sinon from 'sinon'
+import * as vscode from 'vscode'
+import { createTestWindow } from '../../shared/vscode/window'
+import { openSsoPortalLink } from '../../../credentials/sso/model'
+import { assertTelemetry } from '../../testUtil'
+import { CancellationError } from '../../../shared/utilities/timeoutUtils'
+
+describe('openSsoPortalLink', function () {
```

Comment:
Normally I wouldn't suggest writing thorough tests for such a small component but it's good to have more examples of one way to test UI.